.cirrus.yml

```diff
@@ -7,6 +7,7 @@ task:
   install_script:
     - pkg install -y libpcap bash gsed gmake tcpreplay
   configure_script:
+    - chmod a+rw /dev/bpf*
```

Comment:
@clementperon what's the purpose of allowing the user access to the network interfaces instead of using `sudo`? How is this better than using `sudo`? 🤔

Reply:
We explicitly control that only network should be accessible whereas with sudo you don't know what your python will do / touch.

Reply:
Also this will make it easier, in the future we only need to pass the interface name for running ctest, we don't need to tell to use sudo anymore

Reply:
ok sure, but maybe put a comment above every `chmod` so we know we did it for?

I'll definitely forget after 2-3 months... 😃

Reply:
done